### DISCUSSION TITLE
Setting custom fonts and branding

### DISCUSSION THREAD

**Discussion by i5hi** [docmost/docmost#412] [Help / Questions] [answered]
Is there a way to set custom fonts and branding?

**Comment by Philipinho** [comment, marked as answer]
Right now, there are no customization options outside the core design.

Do you have any ideas in mind?

**Reply by i5hi** [reply]
We could move the font selection and logo to a config file which users can customize.

**Reply by i5hi** [reply]
it would be a one time setting that would done on setup or require a restart if done later. Need not be something that is customizable via the GUI.

**Reply by tobifanger** [reply]
> We could move the font selection and logo to a config file which users can customize.

Has it already been implemented? It would be a significant step forward. Background color would be also useful. Thx
